src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -86,15 +86,30 @@ Transactor::Transactor(
 
 void Transactor::calculateFee ()
 {
-    mFeeDue = STAmount (getApp().getFeeTrack().scaleFeeLoad(
+    LoadFeeTrack& lft = getApp().getFeeTrack();
+
+    mFeeDue = STAmount (lft.scaleFeeLoad(
         calculateBaseFee(), view().fees().base,
-            view().fees().units, view().flags() & tapADMIN));
+            lft.getLoadBase(), view().flags() & tapADMIN));
 }
```

Comment:
Scott and I hashed it out, and we have reached a rough consensus that
* `getLoadBase` is not the proper place to get fee units.
* `view().fees().unit` is the proper place.
* `LoadFeeTrack` can be thought of as basing it's operations on a "load unit", but that those units are generally never exposed outside of that class. (eg. In `scaleFeeLoad`'s computations, it multiplies by a "load unit" value, then divides by a "load unit" value, canceling out any unit concerns.)